### DISCUSSION TITLE
How do I build this on Windows?

### DISCUSSION THREAD

**Discussion by asdf32768** [adepierre/Botcraft#97] [Q&A] [answered]
What tools do I need to have to build this on windows and how to do it

**Comment by adepierre** [comment, marked as answer]
Hello !

You can find a precompiled Windows version [here]( https://github.com/adepierre/Botcraft/releases/tag/latest), and a detailed compilation guide [here](https://github.com/adepierre/Botcraft/wiki/Detailed-installation-guide) in case you want to build it yourself.

Feel free to ask if something isn't clear.

**Reply by asdf32768** [reply]
thank you very much
